patches/hideFooter.css

```
@@ -0,0 +1,5 @@
+@media screen and (max-width: 1023px) {
+    footer {
+        display: none;
```

This style is overwriten by page css

```suggestion
        display: none !important;
```